2020-12-29 11:55:06,933 INFO    MainThread:13296 [wandb_setup.py:_flush():70] setting env: {}
2020-12-29 11:55:06,933 INFO    MainThread:13296 [wandb_setup.py:_flush():70] multiprocessing start_methods=spawn
2020-12-29 11:55:06,933 INFO    MainThread:13296 [wandb_setup.py:_flush():70] setting login settings: {}
2020-12-29 11:55:06,934 INFO    MainThread:13296 [wandb_init.py:_log_setup():303] Logging user logs to C:\Users\משתמש\Desktop\everything\magshimim\mag project\camerona\training\wandb\run-20201229_115506-3jjv1nij\logs\debug.log
2020-12-29 11:55:06,934 INFO    MainThread:13296 [wandb_init.py:_log_setup():304] Logging internal logs to C:\Users\משתמש\Desktop\everything\magshimim\mag project\camerona\training\wandb\run-20201229_115506-3jjv1nij\logs\debug-internal.log
2020-12-29 11:55:09,180 INFO    MainThread:13296 [wandb_run.py:_console_start():1237] atexit reg
2020-12-29 11:55:09,181 INFO    MainThread:13296 [wandb_run.py:_redirect():1104] redirect: SettingsConsole.WRAP
2020-12-29 11:55:09,181 INFO    MainThread:13296 [wandb_run.py:_redirect():1135] Wrapping output streams.
2020-12-29 11:55:09,181 INFO    MainThread:13296 [wandb_run.py:_redirect():1151] Redirects installed.
2020-12-29 11:55:27,919 INFO    MainThread:13296 [wandb_run.py:_atexit_cleanup():1207] got exitcode: 1
2020-12-29 11:55:27,919 INFO    MainThread:13296 [wandb_run.py:_restore():1179] restore
2020-12-29 11:55:27,924 INFO    MainThread:13296 [wandb_run.py:_wait_for_finish():1324] got exit ret: uuid: "5efeb195ccd2405abf1def1367b4d3fd"
response {
  poll_exit_response {
    file_counts {
      wandb_count: 1
    }
    pusher_stats {
      uploaded_bytes: 872
      total_bytes: 872
    }
  }
}

2020-12-29 11:55:29,927 INFO    MainThread:13296 [wandb_run.py:_wait_for_finish():1324] got exit ret: uuid: "cc784e4cb5614c19834b19dd37cee7d3"
response {
  poll_exit_response {
    file_counts {
      wandb_count: 5
    }
    pusher_stats {
      uploaded_bytes: 6510
      total_bytes: 6510
    }
  }
}

2020-12-29 11:55:31,931 INFO    MainThread:13296 [wandb_run.py:_wait_for_finish():1324] got exit ret: uuid: "c60979fe69b24519916e364b8bccbdfe"
response {
  poll_exit_response {
    done: true
    exit_result {
    }
    file_counts {
      wandb_count: 5
    }
    pusher_stats {
      uploaded_bytes: 6510
      total_bytes: 6510
    }
  }
}

2020-12-29 11:55:33,056 INFO    MainThread:13296 [wandb_run.py:_show_files():1536] logging synced files
2020-12-29 11:55:33,061 INFO    MainThread:13296 [internal.py:handle_exit():138] Internal process exited
